Packs/DeveloperTools/Integrations/CustomIndicatorDemo/IndicatorTypes/mycustomindicator.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
should be under `Packs/DeveloperTools/IndicatorTypes/mycustomindicator.json`

Reply:
Fixed